pkg/v1/tkg/client/create_test.go

```diff
@@ -115,4 +116,36 @@ spec:
 			})
 		})
 	})
+
+	Describe("WaitForAutoscalerDeployment", func() {
```

Comment:
Nice to have: a test where the TKGConfigReaderWriter returns an error

Reply:
This is the case where TKGConfigReaderWriter returns an error.

https://github.com/vmware-tanzu/tanzu-framework/pull/1398/files#diff-7d1cc721897f32ac26349f1835174055ad258f3665adee85734b23f3f024e269R125